test/test_mkldnn.py

```diff
@@ -231,12 +237,27 @@ def test_linear(self):
         x = torch.randn(3, in_features, dtype=torch.float32) * 10
 
         for bias in [True, False]:
-            linear = torch.nn.Linear(in_features, out_features).float()
+            linear = torch.nn.Linear(in_features, out_features, bias=bias).float()
             mkldnn_linear = mkldnn_utils.to_mkldnn(copy.deepcopy(linear))
             self.assertEqual(
                 linear(x),
                 mkldnn_linear(x.to_mkldnn()).to_dense())
 
+            self._test_serialization(mkldnn_linear, (x.to_mkldnn(),))
+
+    def _test_serialization(self, module, inputs):
+        if IS_WINDOWS or IS_SANDCASTLE:
+            # TemporaryFileName is not supported in Windows and Sandcastle
```

Comment:
there's a helper function for it already in test_jit.py. Maybe pull it out to common_utils.py and use here?